data/ecosystems/i/iotex.toml

```diff
@@ -1,7 +1,7 @@
 # Ecosystem Level Information
 title = "IoTeX"
 
-sub_ecosystems = []
+sub_ecosystems = ["FilDA", "Mimo", "ioTube", "UniFi", "ZoomSwap", "xDollar", "MinMax Finance", "Parrot Defi", "ForTube", "WOWSwap", "BoringDAO", "Magic Land", "Elk", "Cyclone", "StarCrazy"]
```

Comment:
You have to add these sub ecosystem toml files before this will validate.

<img width="859" alt="image" src="https://user-images.githubusercontent.com/41347/216233629-7c52088b-d7c2-4002-b2af-0f02b90a1bf5.png">